clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/WebConfig.java

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
@mattgogerly I saw https://github.com/spinnaker/clouddriver/pull/5703 with some other groovy --> java, but I didn't see this file so I think there's no conflict/race.